README.md

```diff
@@ -66,9 +66,9 @@ Properties in each file override the previous values (by object deep merge).
 
 ### Out-of-the-box presets:
 
--   `-p bootstrap`: Default [preset](https://github.com/nemtech/symbol-bootstrap/blob/main/presets/bootstrap/network.yml). It's a private network with 1 mongo database, 2 peers, 1 api and 1 rest gateway. Nemesis block is generated.
+-   `-p bootstrap`: A [preset](https://github.com/nemtech/symbol-bootstrap/blob/main/presets/bootstrap/network.yml). It's a private network with 1 mongo database, 2 peers, 1 api and 1 rest gateway. Nemesis block is generated.
```

Comment:
shouldn't this be a named preset ?

Reply:
I've updated the docs for this pr. 

I'm planning to change the assemblies, please have a look at the draft pr https://github.com/symbol/symbol-bootstrap/pull/328

Preview doc https://github.com/symbol/symbol-bootstrap/tree/shared_assemblies#preset